### DISCUSSION TITLE
submit button remains disabled after upload file

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#2357] [Help]
When i try to update a record with an upload file element, the file upload right but the button Submit remains disabled:

**Comment by danharrin** [comment]
Make sure you're on the latest version :)

**Reply by jcgmalave** [reply]
I'm on the latest version, I think the error is caused by the fact that the uploaded file input is inside a hasmanyrepeater

**Reply by danharrin** [reply]
Please submit an issue with reproducible code
